code/scripting/api/objs/briefing.cpp

```diff
@@ -0,0 +1,198 @@
+
+#include "briefing.h"
+
+namespace scripting {
+namespace api {
+
+//**********HANDLE: briefing
+ADE_OBJ(l_BriefStage, brief_stage, "briefing_stage", "Briefing stage handle");
+
+ADE_VIRTVAR(Text, l_BriefStage, nullptr, "The text of the stage", "briefing_stage", "The text")
```

Comment:
The fifth argument here is the return type and should be "string" and not "briefing_stage"

Reply:
Another instance where I duplicated what was done in cmd_brief.cpp. However, this one I'm understanding. Looks like cmd_brief.cpp will need a slight cleanup to match. I'll add it to me to-do list for the API.

Reply:
Yes, it'll likely also need changing in cmd_brief
Must have slipped though